jupyterlab_server/config.py

```diff
@@ -53,10 +53,17 @@ def get_federated_extensions(labextensions_path):
     return federated_extensions
 
 
-def get_static_page_config(app_settings_dir=None, logger=None):
+def get_static_page_config(app_settings_dir=None, logger=None, all_levels=False, user=False, sys_prefix=False):
```

Comment:
all_levels, user, and sys_prefix are mutually exclusive here, right? Is it more elegant to have a single parameter that can take three values, with a single default value?